clients/banking/src/components/CardItemArea.tsx

```diff
@@ -110,10 +110,19 @@ export const CardItemArea = ({
 
   const isCurrentUserCardOwner = userId === card?.accountMembership.user?.id;
 
+  const membershipStatus = card?.accountMembership.statusInfo;
+
   const cardRequiresIdentityVerification =
-    B2BMembershipIDVerification === true && card?.accountMembership.statusInfo.status !== "Enabled";
+    B2BMembershipIDVerification === true &&
+    membershipStatus?.__typename === "AccountMembershipBindingUserErrorStatusInfo" &&
+    membershipStatus.idVerifiedMatchError;
 
-  const bindingUserError = card?.accountMembership.statusInfo.status === "BindingUserError";
+  const bindingUserError =
+    membershipStatus?.__typename === "AccountMembershipBindingUserErrorStatusInfo" &&
+    (membershipStatus.birthDateMatchError ||
+      membershipStatus.firstNameMatchError ||
+      membershipStatus.lastNameMatchError ||
+      membershipStatus.phoneNumberMatchError);
```

Comment:
small question: should we care about this kind of binding error when `B2BMembershipIDVerification` is `false`? does it prevent card manipulations?

Reply:
will ask a product manager to be sure